### ISSUE TITLE
Vector database or Store embeddings in existing Antenna database (with logits, etc)

### ISSUE THREAD

**Issue by mihow** [RolnickLab/antenna#776]

**Comment by mihow** [comment]
Duplicate of #752
